.github/workflows/tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Tests
+on:
+  pull_request:
+    types: [labeled, unlabeled, opened, edited, synchronize]
+
+jobs:
+  base_setup:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ["3.10"]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
It seems like this is running the setup for 3 different platforms. Is this required for lint check?